### ISSUE TITLE
XAML Controls Gallery: Accessibility: [MAS4.3.1]: In HC mode, list items in multiple cards are not visible properly when selected.

### ISSUE THREAD

**Issue by v-amnat** [microsoft/microsoft-ui-xaml#4276]
Tags
Accessibility, A11yCT, A11yMAS, A11yMediumImpact, MAS4.3.1, HighContrast, Sev3, XAML Controls Gallery (Debug), Win32, Wipro

**Steps to reproduce the bug**

1.Launch the XAML Controls Gallery (Debug) App
2.Go to the Whats new-->List box-->Navigate to list items and observe
Options are not visible properly.

**Note**
Same is observed in Compact sizing, ParallexView cards as well

**Actual behavior**
In HC mode, list items in multiple cards are not visible properly when selected.
 
**Expected behavior**
Everything should visible properly in HC mode.

**User Impact**
People with Low vision are not able to identify the selected option  in the list in multiple catds.

**Version Info**
Microsoft OS: 21313.1000
App: XAML Controls Gallery 
![parallexview hcb](https://user-images.githubusercontent.com/74585893/108680844-5db8d500-74bc-11eb-9638-ac5e5f43f51f.png)
![list boc hcb](https://user-images.githubusercontent.com/74585893/108680887-6ad5c400-74bc-11eb-8491-8b13d4ca5855.png)
![listbox hcw](https://user-images.githubusercontent.com/74585893/108680914-71fcd200-74bc-11eb-831f-855fa7b23815.png)
![compact sizing hcb](https://user-images.githubusercontent.com/74585893/108680940-7c1ed080-74bc-11eb-99cc-3d7c537a40f9.png)
![compact sizing hcw](https://user-images.githubusercontent.com/74585893/108680942-7cb76700-74bc-11eb-87e6-8e693b0d681c.png)
